src/components/swimlanes/swimLanes.tsx

```diff
@@ -0,0 +1,78 @@
+import {
+  Box,
+  Card,
+  CardContent,
+  CardHeading,
+  Flex,
+  Heading,
+  SimpleGrid,
+  Spacer,
+  Text,
+  Link,
+} from "@nypl/design-system-react-components";
+import styles from "./Swimlanes.module.css";
+import React from "react";
+import { imageURL } from "@/utils/utils";
+
+const SwimLanes = ({ lanesWithNumItems }) => {
+  return (
+    <>
+      {lanesWithNumItems.map((lane, key) => (
+        <Box data-testid={lane.slug} key={lane.rank} mb="l">
+          <Flex alignItems="baseline">
+            <Heading id={`row-heading-${lane.slug}`} level="h2" size="heading3">
+              {lane.title}
+            </Heading>
+            <Spacer />
+            <Link
+              id={`row-see-more-${lane.slug}`}
+              type="standalone"
+              href={`${process.env.DC_URL}collections/lane/${lane.slug}`}
```

Comment:
There are some ways to encapsulate the env variables so you don't have to type `process.env` within react components. One suggestion is to create an `appConfig.ts` file that contains all your env variables. Then you can import it in this file and not have `process.env`. 
```
// appConfig.ts

export {
  DC_URL: process.env.DC_URL,
  ...
};
```

Reply:
I find it most helpful for organization so you can see all the env vars in one config file. Once you start adding more variables, it will also be helpful see the categorization of server-side only vars (e.g. for credentials) and client-side only vars that have the `NEXT_PUBLIC_` prefix.